runtime/compiler/control/J9Options.cpp

```diff
@@ -418,16 +418,16 @@ qualifiedSize(UDATA *byteSize, char **qualifier)
    UDATA size;
 
    size = *byteSize;
-   *qualifier = "";
+   *qualifier = (char *)"";
```

Comment:
It looks like the only uses of the function `qualifiedSize` appear in [`J9::Options::preProcessCodeCachePrintCodeCache`](https://github.com/eclipse-openj9/openj9/blob/e18f20d856f9a71a4341bc3fdc88066e7d1a23e1/runtime/compiler/control/J9Options.cpp#L2006-L2014) below.  If I'm understanding correctly, I think the arguments that are passed to this function could be declared as `const char *`, and so `qualifier` could be declared to be `const char **`.  If so, that would avoid all these casts in assigning to `*qualifier`.